gst/nnstreamer/tensor_if/gsttensorif.c

```diff
@@ -130,3 +133,243 @@ static gboolean gst_tensor_if_start (GstBaseTransform * trans);
 static gboolean gst_tensor_if_stop (GstBaseTransform * trans);
 static gboolean gst_tensor_if_sink_event (GstBaseTransform * trans,
     GstEvent * event);
+
+static void gst_tensor_if_install_properties (GObjectClass * gobject_class);
+
+/**
+ * @brief initialize the tensor_if's class (GST Standard)
+ */
+static void
+gst_tensor_if_class_init (GstTensorIfClass * klass)
+{
+  GObjectClass *gobject_class;
+  GstElementClass *gstelement_class;
+  GstBaseTransformClass *trans_class;
+
+  GST_DEBUG_CATEGORY_INIT (gst_tensor_if_debug, "tensor_if", 0,
+      "Tensor if to control streams based on tensor(s) values");
+
+  trans_class = (GstBaseTransformClass *) klass;
+  gstelement_class = (GstElementClass *) trans_class;
+  gobject_class = (GObjectClass *) gstelement_class;
+
+  gobject_class->set_property = gst_tensor_if_set_property;
+  gobject_class->get_property = gst_tensor_if_get_property;
+  gobject_class->finalize = gst_tensor_if_finalize;
+
+  gst_tensor_if_install_properties (gobject_class);
+
+  gst_element_class_set_details_simple (gstelement_class,
+      "Tensor_If",
+      "NNStreamer/If",
+      "Controls streams based on the tensor(s) values",
+      "MyungJoo Ham <myungjoo.ham@samsung.com>");
+
+  gst_element_class_add_pad_template (gstelement_class,
+      gst_static_pad_template_get (&src_factory));
+  gst_element_class_add_pad_template (gstelement_class,
+      gst_static_pad_template_get (&sink_factory));
+
+  /* Refer: https://gstreamer.freedesktop.org/documentation/design/element-transform.html */
```

Comment:
Refer link seems changed to  https://gstreamer.freedesktop.org/documentation/additional/design/element-transform.html

Reply:
URL updated.